Avoid conflicts between sys/mount.h and linux/mount.h

linux/fs.h includes linux/mount.h and this include file is unused so
do not include it and avoid conflict too with glibc 2.36+ see [1]

[1] https://sourceware.org/glibc/wiki/Release/2.36#Usage_of_.3Clinux.2Fmount.h.3E_and_.3Csys.2Fmount.h.3E

Upstream-Status: Pending

Signed-off-by: Khem Raj <raj.khem@gmail.com>
--- a/modules/parse_amd.c
+++ b/modules/parse_amd.c
@@ -27,7 +27,6 @@
 #include <sys/utsname.h>
 #include <netinet/in.h>
 #include <sys/mount.h>
-#include <linux/fs.h>
 
 #define MODULE_PARSE
 #include "automount.h"
--- a/modules/parse_sun.c
+++ b/modules/parse_sun.c
@@ -30,7 +30,6 @@
 #include <sys/utsname.h>
 #include <netinet/in.h>
 #include <sys/mount.h>
-#include <linux/fs.h>
 
 #define MODULE_PARSE
 #include "automount.h"
